src/webinars/2024/bringing-node-red-to-industrial-solutions-with-wago.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Bringing Node-RED to Industrial Solutions with Wago"
+subtitle: Nick O'Leary, Co-Creator of Node-RED & CTO at FlowFuse, and Kurt Braun, Director of Automation Sales Engineering and Development at Wago, discuss how to bridge Node-RED to industrial solutions with Wago.
```

Comment:
@Hasmin-AC I wasn't sure what to add here, I made it up following the structure of a past webinar, but I think it could be improved.